addon/-private/data-view/radar/radar.js

```diff
@@ -207,24 +209,33 @@ export default class Radar {
     assert('itemContainer must be set on Radar before scheduling an update', itemContainer !== null);
     assert('scrollContainer must be set on Radar before scheduling an update', scrollContainer !== null);
 
-    const {
-      top: itemContainerTop
-    } = itemContainer.getBoundingClientRect();
-    const {
-      top: scrollContainerTop,
-      height: scrollContainerHeight
-    } = scrollContainer.getBoundingClientRect();
+    // The scroll container's offsetHeight will reflect the actual height of the element, while
+    // it's measured height via bounding client rect will reflect the height with any transformations
+    // applied. We use this to find out the scale of the items so we can store measurements at the
+    // correct heights.
+    const scrollContainerOffsetHeight = scrollContainer.offsetHeight;
+    const { height: scrollContainerRenderedHeight } = scrollContainer.getBoundingClientRect();
+
+    // Represents the opposite of the scale, if any, applied to the collection
+    const transformScale = scrollContainerOffsetHeight > 0 ? scrollContainerOffsetHeight / scrollContainerRenderedHeight : 1;
```

Comment:
I'm unsure but this may require a guard against floating point errors

Reply:
Yeah, I thought about that, unsure if rounding would be better or worse in the end. I can try to come up with some tests for it